tools/gen_addon_readme.template

```diff
@@ -11,10 +11,13 @@
 {{ manifest.name }}
 {{ '=' * manifest.name|length }}
 
-.. !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+.. 
```

Comment:
Why this extra line?

Reply:
Because this is the [correct syntax for multiline rst comments](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#comments).